Sources/TuistKit/Commands/BuildCommand.swift

```diff
@@ -91,12 +115,50 @@ public struct BuildCommand: AsyncParsableCommand {
     @OptionGroup()
     var buildOptions: BuildOptions
 
+    private var notAllowedPassthroughXcodeBuildArguments = [
+        "-scheme",
+        "-workspace",
+        "-project",
+    ]
+
     public func run() async throws {
-        let absolutePath: AbsolutePath
-        if let path = buildOptions.path {
-            absolutePath = try AbsolutePath(validating: path, relativeTo: FileHandler.shared.currentPath)
+        // Check if passthrough arguments are already handled by tuist
+        try notAllowedPassthroughXcodeBuildArguments.forEach {
+            if buildOptions.passthroughXcodeBuildArguments.contains($0) {
+                throw XcodeBuildPassthroughArgumentError.alreadyHandled($0)
```

Comment:
I think we should only raise in the case of `-scheme`, when scheme has already been provided via `tuist build/test {scheme}`. Otherwise, we should accept it as a valid passthrough argument. Thoughts?

Reply:
The concern I have is that when no scheme is provided as argument but as passthrough argument, a buildable entry scheme would be [inferred](https://github.com/tuist/tuist/blob/main/Sources/TuistKit/Services/BuildService.swift#L154), added later to `xcodebuild` [here](https://github.com/tuist/tuist/blob/main/Sources/TuistAutomation/XcodeBuild/XcodeBuildController.swift#L52). 

To allow this use case, it would require inspecting (around [here](https://github.com/tuist/tuist/blob/main/Sources/TuistKit/Services/BuildService.swift#L102)) the passthrough arguments to avoid the scheme being inferred and passed twice to `xcodebuild`, hence throwing.

This is my insight, but I leave to other people with broader knowlodge to make the call.